shared/django_apps/codecov_auth/models.py

```diff
@@ -149,6 +151,37 @@ def __str__(self):
         str_representation_of_is_active = "Active" if self.is_active else "Inactive"
         return f"{str_representation_of_is_active} Account: {self.name}"
 
+    def activate_user_onto_account(self, user: User) -> None:
+        self.users.add(user)
+        self.save()
+
+    def activate_owner_user_onto_account(self, owner_user: "Owner") -> None:
+        user: User = owner_user.user
+        if not user:
+            user = User(name=user.name, email=user.email)
+        self.activate_user_onto_account(user)
+
+    def deactivate_owner_user_from_account(self, owner_user: "Owner") -> None:
+        if owner_user.user is None:
+            return
+
+        organizations_in_account: list[Owner] = self.organizations.all()
+        all_users_for_account: list[AutoField] = [
```

Comment:
[nit] this probably doesn't matter for most cases, but since we're checking for membership of this list later and not iterating then I think if AutoField is hashable a set might be a better data structure to use here